### PR TITLE
Added email and password fields to the edit profile form

### PR DESCRIPTION
**Pull request by VABIII**
## Description

Added email and password fields to the edit profile form

#### Video Link

[Loom Video](https://www.loom.com/share/b39578a0df1c4d7784a27d816f1b9df2)

#### Trello Link

<blockquote class="trello-card"><a href="https://trello.com/c/ICcTAQkL/166-as-a-mentor-mentee-or-admin-i-want-to-update-my-password-and-email-in-account-settings">Fix: Added additional fields to the edit profile form to accommodate requested additions </a></blockquote>

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have removed unnecessary comments/console logs from my code
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings
- [ ] No duplicate code left within changed files
- [ ] Size of pull request kept to a minimum
- [ ] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by AdamHVu — APPROVED** [review]
Cool new feature that Im sure will be great once fixed

File: src/components/pages/Profile/EditProfile.js [hunk 1]
```diff
@@ -145,6 +145,19 @@ function EditProfile({ userInfo, setUserProfile }) {
             <Input />
           </Form.Item>
 
+          <Form.Item
```
Comment (kimnivore) [inline src/components/pages/Profile/EditProfile.js:148]: Nice work using antd to align the password form item.

**Review by kimnivore — APPROVED** [review]

**Review by yuririnnnu — APPROVED** [review]
The changes look good to me. Thanks.

**Review by sebas6292 — COMMENTED** [review]
Solid clean code. Is it possible to add a placeholder or would that conflict with antd design?

**Pull request merged**